example/index.js

```diff
@@ -1,11 +1,56 @@
 'use strict';
-
 const React = require('react');
 const ReactDOM = require('react-dom');
+const { Tooltip, Textfield, Switch, Spinner, Slider, Radio, Progress, IconToggle, Button, Badge, Checkbox } = require('../');
+
+const exampleStyle = {
+  margin: 'auto',
+  width: '800',
+  border: '1px solid black',
+  padding: '10px'
+}
 
 const exampleTree = (
-  <div>
-    Examples
+  <div style={exampleStyle}> 
+    <h1>Examples</h1>
+
+    <h4>Badges</h4>
+      <Badge data-badge="2">Badge</Badge>
+
+    <h4>Buttons</h4>
+      <Button>Test Button</Button>
+
+    <h4>Checkboxes</h4>
+      <Checkbox />
+
+    <h4>Icon Toggle</h4>
+      <IconToggle ripple id='2' label='test' />
```

Comment:
Add another IconToggle without `id` to test generation and make the label "Without id".  Change the label of this one to "With id"